package.json

```diff
@@ -72,7 +72,7 @@
     "webpack-node-externals": "1.7.2"
   },
   "peerDependencies": {
-    "@ahaui/css": "^2.0.0",
+    "@ahaui/css": "2.0.4-alpha.7",
```

Comment:
@khuctrang peerDependency shouldn't be a fixed version. Try `"@ahaui/css": "^2"` instead